README.md

```diff
@@ -2,15 +2,16 @@
 
 Installation:
 
-  * Make sure you have **Elixir** and **Phoenix** installed and up-to-date
-  * Install dependencies with `mix deps.get`
+  * Make sure you have **Elixir** and **Node** installed and up-to-date
+  * Install Elixir dependencies with `mix deps.get`
   * Install Node.js dependencies with `mix npm install`
   * Install Elm dependencies with `mix elm install`
-  * Optional: install linters with `npm install -g eslint jshint sass-lint elm`
+  * Optional: install linters with `npm install -g eslint sass-lint elm`
 
 Usage:
+
   * Run the up with `mix phx.server`
-  * Run the test with `mix test`
+  * Run the specs with `mix test` and `mix elm test`
```

Comment:
I will prefer to have consistency in names.
We should use test or specs, but not both

Reply:
Done